<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyGen</class>
 <widget class="QDialog" name="KeyGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Encryption Key</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Key Size (MiB)</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderKeySize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>391</width>
     <height>18</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="keyGenButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate and Save ...</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="keySize">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File Path:</string>
   </property>
  </widget>
  <widget class="QLabel" name="filePath">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>871</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>   </string>
   </property>
  </widget>
  <action name="actionkeyGenButton">
   <property name="text">
    <string>keyGenButton</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
